src/response.rs

```diff
@@ -12,4 +13,14 @@ pub struct ValidationResponse {
     pub code: Option<u16>,
     /// Mutated Object - used only by mutation policies
     pub mutated_object: Option<serde_json::Value>,
+    /// AuditAnnotations is an unstructured key value map set by remote admission controller (e.g. error=image-blacklisted).
+    /// MutatingAdmissionWebhook and ValidatingAdmissionWebhook admission controller will prefix the keys with
+    /// admission webhook name (e.g. imagepolicy.example.com/error=image-blacklisted). AuditAnnotations will be provided by
+    /// the admission webhook to add additional context to the audit log for this request.
+    pub audit_annotations: Option<HashMap<String, String>>,
```

Comment:
Are these annotations supposed to be used with OpenTelemetry traces ? Or are they available in other places as well?

Reply:
They are shown inside of the kubernetes audit log